src/main/app/controller/courts/CourtTypesController.ts

```diff
@@ -10,10 +10,10 @@ import {CSRF} from '../../../modules/csrf';
 export class CourtTypesController {
 
 
-  emptyCourtCodeErrorMsg = 'Court code is required and must be numeric.';
+  emptyCourtCodeErrorMsg = 'Court code is required and must be numeric and start with 1-9';
   getCourtTypesErrorMsg = 'A problem occurred when retrieving the list of court types.';
-  updateErrorMsg = 'A problem occurred when saving the court court types.';
-  emptyCourtTypesErrorMsg = 'One or more court types are required.';
+  updateErrorMsg = 'A problem occurred when saving the court types.';
+  emptyCourtTypesErrorMsg = 'One or more court types are required for types entries.';
   private magistrateCourtType = "Magistrates' Court";
   private countyCourtType = 'County Court';
   private crownCourtType = 'Crown Court';
```

Comment:
Could this be replaced with an enum?

```
enum courtType {
  magistrate = "Magistrates' Court",
  county = 'County Court,
  crown = 'Crown Court'
  family = 'Family Court',
  tribunal = 'Tribunal'
}
```

Apologies if any of those strings are wrong.

Reply:
Changed to use enum for the 3 court types that have a code associated.